pvlib/pvsystem.py

```diff
@@ -421,6 +421,87 @@ def sapm_effective_irradiance(self, poa_direct, poa_diffuse,
             poa_direct, poa_diffuse, airmass_absolute, aoi,
             self.module_parameters, reference_irradiance=reference_irradiance)
 
+    def first_solar_spectral_loss(self, pw, airmass_absolute):
+        
+        """
+        Use the :py:func:`first_solar_spectral_correction` function to 
+        calculate the spectral loss modifier.
```

Comment:
Doc string should have some kind of reference to ``self.module_parameters``. Most of the other ``PVSystem`` method doc strings have some examples. This method is a little more complicated though, so might be worth being more explicit about what is looked for. For example...

Use the :py:func:`first_solar_spectral_correction` function to calculate the spectral loss modifier. The spectral loss modifier is determined by searching for one of the following keys in ``self.module_parameters`` (in order): ``'first_solar_spectral_coefficients'``, ``'Technology'``, ``'Material'``.

Reply:
Use the :py:func:`first_solar_spectral_correction` function to
        calculate the spectral loss modifier. The model coefficients are
        specific to the module's cell type, and are determined by searching
        for one of the following keys in self.module_parameters (in order):
            'first_solar_spectral_coefficients (user-supplied coefficients)
            'Technology' - a string describing the cell type, can be read from
            the CEC module parameter database
            'Material' - a string describing the cell type, can be read from
            the Sandia module database.